### ISSUE TITLE
upgrade to 9.0.2 from 8.4.1 exception

### ISSUE THREAD

**Issue by sogehige** [sogehige/sogeBot#2166]
```
2019-05-03T02:33:29.187 !!! ERROR !!! Possibly Unhandled Rejection at: Promise {
  <rejected> TypeError: Cannot read property 'isSystemEnabled' of undefined
      at new CommercialWidget (/home/user/sogebot/src/bot/widgets/commercial.js:3:8)
      at Object.<anonymous> (/home/user/sogebot/src/bot/widgets/commercial.js:13:84)
      at Module._compile (internal/modules/cjs/loader.js:816:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
      at Module.load (internal/modules/cjs/loader.js:685:32)
      at Function.Module._load (internal/modules/cjs/loader.js:620:12)
      at Module.require (internal/modules/cjs/loader.js:723:19)
      at require (internal/modules/cjs/helpers.js:14:16)
      at require (/home/user/sogebot/src/bot/commons.ts:18:22)
      at autoLoad (/home/user/sogebot/src/bot/main.js:93:28) } reason: TypeError: Cannot read property 'isSystemEnabled' of undefined
```

**Comment by sogehige** [comment]
I am not able to reproduce

**Comment by MoonlightAshe** [comment]
The issue is that I cannot connect to the panel and this is the only error that is shown when starting up the bot.
So I tried to do incremental updates from 8.4.1 to 9.0.2 starting with 9.0.0 but I get the same error and the same problem. This happens in both Linux and Windows, and even tried it fresh instead of upgrading with same result. 
In the meantime, I went back to 8.4.1 which still works for me.
![image](https://user-images.githubusercontent.com/25715993/57343548-64206600-7109-11e9-9c9c-4cfd42a6b2bc.png)
And continues...
2019-05-07T20:46:21.169 | ENABLED: Raffles (systems)
2019-05-07T20:46:21.171 | ENABLED: Ranks (systems)
2019-05-07T20:46:21.174 | ENABLED: Scrim (systems)
2019-05-07T20:46:21.174 | DISABLED: Songs (systems)
2019-05-07T20:46:21.175 | ENABLED: Timers (systems)
2019-05-07T20:46:21.178 | ENABLED: Top (systems)
2019-05-07T20:46:21.179 | ENABLED: UserInfo (systems)
2019-05-07T20:46:24.617 | CURRENCY: fetching rates
2019-05-07T20:46:24.867 | CURRENCY: fetched rates
Doesn't go beyond this ^
